.prettierrc

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
this is the one we use https://github.com/MetaMask/metamask-module-template/blob/main/.prettierrc.js

regarding changes to changelog, it was fixed here: https://github.com/MetaMask/metamask-module-template/commit/09cba70f2dcc776ff008e32c63cfdc85effd9551

Reply:
Thank you so much! I was struggling with linting vs prettier 😂 !